css/global.css

```diff
@@ -63,6 +129,54 @@ img {
 }
 
 /* Text */
+h1 {
+	font-size: var(--font56);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height58);
+	color: var(--color-white);
+}
+
+h2 {
+	font-size: var(--font46);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height50);
+	color: var(--color-onyx);
+}
+
+h3 {
+	font-size: var(--font36);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height40);
+	color: var(--color-white);
+}
+
+h4 {
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+	color: var(--color-white);
+}
+
+h5 {
+	font-size: var(--font24);
+	font-weight: var(--font-weight-medium);
+	line-height: var(--line-height28);
+	color: var(--color-onyx);
+}
+
+p {
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	line-height: var(--line-height20);
+	color: var(--color-white);
+}
+
+.body1 {
```

Comment:
can you please describe the use of body 1 with comments?

Reply:
We sure did! Let us know if this was descriptive enough!